pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -63,6 +64,15 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return nil
 	}
 
+	// Refreshing cache info about target OS
+	isSUSE, err := target.IsSUSEOS()
+	if err != nil {
+		return err
+	}
+	// Check if target node OS is valid
+	if !isSUSE {
+		log.Println("Invalid target node OS")
```

Comment:
Just one small thing, sorry I didn't catch it before, can you change `log` with `klog` like here: https://github.com/SUSE/skuba/blob/master/cmd/skuba/cluster/init.go#L54 that way we maintain standard logs.

Reply:
:+1: